name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/dialogs/DisplayTextDialog.java

```diff
@@ -103,12 +122,134 @@ protected void createButtonsForButtonBar(Composite parent)
     protected Control createDialogArea(Composite parent)
     {
         Composite container = new Composite(parent, SWT.None);
-        GridDataFactory.fillDefaults().grab(true, true).hint(600, 200).applyTo(container);
+        GridDataFactory.fillDefaults().grab(true, true).hint(800, 500).applyTo(container);
         GridLayoutFactory.fillDefaults().numColumns(1).applyTo(container);
 
+        if (dialogTitle != null)
+            getShell().setText(dialogTitle);
+
+        if (additionalText != null)
+        {
+            List<StyleRange> styles = new ArrayList<>();
+            addBoldFirstLine(additionalText, styles);
+            additionalText = addMarkdownLikeHyperlinks(additionalText, styles);
+
+            StyledText additionalTextBox = new StyledText(container, SWT.MULTI | SWT.NONE | SWT.WRAP | SWT.READ_ONLY);
+            additionalTextBox.setText(additionalText);
+            additionalTextBox.setStyleRanges(styles.toArray(new StyleRange[0]));
+
+            additionalTextBox.addListener(SWT.MouseDown, e -> openBrowser(e, additionalTextBox));
+
+            container.setBackground(Display.getDefault().getSystemColor(SWT.COLOR_WHITE));
+            GridDataFactory.fillDefaults().grab(true, false).hint(SWT.DEFAULT, SWT.DEFAULT).align(SWT.FILL, SWT.TOP)
+                            .applyTo(additionalTextBox);
+            additionalTextBox.setLayoutData(new GridData(SWT.FILL, SWT.TOP, true, false, 1, 1));
+            additionalTextBox.setMargins(5, 5, 5, 5);
```

Comment:
GridDataFactory is setting the layoutdata (in "applyTo"). Then it is overwritten one line later.